Neos.Utility.MediaTypes/Scripts/updatemediatypes.php

```diff
@@ -20,7 +20,7 @@
 
 $mediaTypesClassPathAndFilename = __DIR__ . '/../Classes/MediaTypes.php';
 
-$rawList = file_get_contents('http://svn.apache.org/viewvc/httpd/httpd/branches/2.4.x/docs/conf/mime.types?revision=HEAD&view=co');
+$rawList = file_get_contents('https://svn.apache.org/viewvc/httpd/httpd/branches/2.4.x/docs/conf/mime.types?revision=HEAD&view=co');
```

Comment:
not sure if the trunk is more up to date ?

Reply:
Was thinking about this too, but apache is slowly going with versions and I'm not sure about their branching policy. Decided to stay with 2.4.x but can be changed ofc.